Python/sysmodule.c

```diff
@@ -1128,18 +1167,20 @@ sys_getwindowsversion(PyObject *self)
 
 #pragma warning(pop)
 
-PyDoc_STRVAR(enablelegacywindowsfsencoding_doc,
-"_enablelegacywindowsfsencoding()\n\
-\n\
-Changes the default filesystem encoding to mbcs:replace for consistency\n\
-with earlier versions of Python. See PEP 529 for more information.\n\
-\n\
-This is equivalent to defining the PYTHONLEGACYWINDOWSFSENCODING\n\
-environment variable before launching Python."
-);
+/*[clinic input]
+sys._enablelegacywindowsfsencoding
+
+Changes the default filesystem encoding to mbcs:replace for consistency with earlier versions of Python.
```

Comment:
Too long line. I suggest to add a dot after "mbcs:replace". And reword the rest as a separate sentence.